docs/sla.md

```diff
@@ -0,0 +1,74 @@
+# Service Level Agreement (SLA)
+
+In production the Covered Service will provide a minimum Biweekly Uptime
+Percentage determined by the Service Level Objective (SLO). If the Covered
+Service does not meet the SLO then a Stability Work Period should be assigned
+to engineers supporting the Covered Service.
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+
+- [Service Level Objective](#service-level-objective)
+- [Definitions](#definitions)
+- [Exclusions](#exclusions)
+- [Additional Information](#additional-information)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Service Level Objective
+
+| Covered Service | Biweekly Uptime Percentage |
+|-----------------|----------------------------|
+| ingestion-edge  | 99.99%                     |
+| ingestion-beam  | 99.5%                      |
+
+## Definitions
+
+**"Covered Service"** means ingestion-edge and ingestion-beam jobs.
+
+**"Downtime"** on ingestion-edge means for at least 0.1% of requests [the
+spec](edge.md) was not followed or a status code lower than 500 was not
+successfully returned. On ingestion-beam it means the spec was not followed for
+at least 0.1% of messages or `oldest_unacked_message_age` on a PubSub input
+exceeds 1 hour for a batch sink job or 1 minute for a decoder or streaming sink
+job.
+
+**"Downtime Period"** means a period of 60 consecutive seconds of Downtime.
+Intermittent Downtime for a period of less than 60 consecutive seconds will not
+be counted towards any Downtime Periods.
+
+**"Biweekly Uptime Percentage"** means total number of minutes in two weeks,
+minus the number of minutes of Downtime suffered from all Downtime Periods in
+two weeks, divided by the total number of minutes in two weeks.
```

Comment:
Is this a rolling two weeks, or are we going to have some defined start date and end date for each biweekly period?

Reply:
rolling period. i'll update this to say that.